Lib/test/test_pathlib.py

```diff
@@ -1693,10 +1693,15 @@ def _check(glob, expected):
                 "dirA", "dirA/linkC", "dirB", "dirB/linkD", "dirC",
                 "dirC/dirD", "dirE", "linkB",
             ])
+        _check(p.rglob(""), ["", "dirA", "dirB", "dirC", "dirE", "dirC/dirD"])
```

Comment:
```suggestion
        _check(p.rglob(""), ["dirA", "dirB", "dirC", "dirE", "dirC/dirD"])
```

Reply:
No, it should include the original directory for compatibility with the old code of `Path.rglob()` and with `glob.glob(recursive=True)`.